### PR TITLE
Add STES Rocketry 

### PR DESCRIPTION
**Pull request by aasitvora99**

### CODE REVIEW

**Comment by Lars-0** [comment]
Hi there!
Thanks for your contribution!
I checked out the website, and saw you are accepting sponsorships, but what is the process for making a donation? Is there a Paypal account?

**Comment by aasitvora99** [comment]
Yes, we do have a PayPal account. I've updated it in the website. You may find it under the sponsor bubble in [Get involved](https://stesrocketry.in/get-involved).

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This is a collection of links and cannot guarantee that all information is up to
...
 | Olin Rocketry      | Olin College of Engineering                               | Spaceport America Cup, Solid Rocket Engine Development                       | [https://www.olinrocketry.com/donate](https://www.olinrocketry.com/donate) | [https://olinrocketry.com/](https://olinrocketry.com/) | [@olinrocketry](https://www.instagram.com/olinrocketry/) |
-
+| STES Rocketry      | Sinhgad Technical Education Society, Pune-India                               | Spaceport America Cup, EuRoC, L2 Solid Rocket Engine Development, L3 Hybrid Engine Development, Plant-based composites, STEM outreach for geographically challenged schools.                        | [https://stesrocketry.in/get-involved](https://stesrocketry.in/get-involved) | [https://stesrocketry.in](https://stesrocketry.in) | [@stesrocketry](https://www.instagram.com/stesrocketry/) |
```
Comment (Lars-0) [inline README.md:34]: Please add 'https://www.paypal.com/paypalme/stesrocketry?country.x=IN&locale.x=en_GB' to the 'How to Donate' column

**Comment by Lars-0** [comment]
Great!
Please add the paypal link (https://www.paypal.com/paypalme/stesrocketry?country.x=IN&locale.x=en_GB) in the 'how to donate' column and I will merge in the pull request.

**Comment by aasitvora99** [comment]
Made the update. Please review.

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ This is a collection of links and cannot guarantee that all information is up to
...
 | Olin Rocketry      | Olin College of Engineering                               | Spaceport America Cup, Solid Rocket Engine Development                       | [https://www.olinrocketry.com/donate](https://www.olinrocketry.com/donate) | [https://olinrocketry.com/](https://olinrocketry.com/) | [@olinrocketry](https://www.instagram.com/olinrocketry/) |
-
+| STES Rocketry      | Sinhgad Technical Education Society, Pune-India                               | Spaceport America Cup, EuRoC, L2 Solid Rocket Engine Development, L3 Hybrid Engine Development, Plant-based composites, STEM outreach for geographically challenged schools.                        | [PayPal: ](https://www.paypal.com/paypalme/stesrocketry?country.x=IN&locale.x=en_GB) | [https://stesrocketry.in](https://stesrocketry.in) | [@stesrocketry](https://www.instagram.com/stesrocketry/) |
```
Comment (Lars-0) [inline README.md:34]: I attempted to verify the paypal link by sending money but received an error message. 
"Sorry, this recipient can’t accept payments right now."
Let me know when this is resolved and once the Paypal account is able to accept money I will merge this in.

Comment (Lars-0) [reply README.md:34]: Hi again, 
I'm checking in to see if things have been resolved regarding accepting payments. Should I give it another try?

**Comment by Lars-0** [comment]
This issue has not been resolved, so I am going to close this pull request. 
If you are able to get a working donation link for your team working, I would, be happy to have you listed. Please make another pull request when ready. 🚀

**Pull request closed without merging**